indico/modules/events/surveys/controllers/management.py

```diff
@@ -164,6 +166,7 @@ class RHOpenSurvey(RHManageSurvey):
     def _process(self):
         if self.survey.state == SurveyState.finished:
             self.survey.end_dt = None
+            self.survey.start_notification_sent = False
```

Comment:
However in that case, the survey will not be "opened" directly, so the code from this `RH` will not be executed.